en/components/input-group.md

```diff
@@ -131,6 +131,142 @@ The Ignite UI for Angular Input Group component can be used inside strictly type
            iframe-src="{environment:demosBaseUrl}/data-entries/typed-form" >
 </code-view>
 
+## Validation
+The following samples demonstrate how input validation could be achieved when using template-driven or reactive forms.
+
+### Template-Driven Forms
+Template-driven form validation is achieved by adding validation attributes, i.e., `required`, `minLength`, etc., to the `input` element.
+
+```html
+<igx-input-group>
+    <label igxLabel for="username">Username</label>
+    <input igxInput name="username" type="text" required />
+</igx-input-group>
+
+<igx-input-group>
+    <label igxLabel for="email">Email</label>
+    <input igxInput name="email" type="email" required email />
+</igx-input-group>
+
+<igx-input-group>
+    <label igxLabel for="password">Password</label>
+    <input igxInput name="password" type="password" required minlength="8" />
+</igx-input-group>
+```
+
+The `required` attribute adds an asterisk next to the label, indicating that this field must be completed, however, when having additional validation like `email` and `minLength`, the user could be notified via the [`igx-hint`]({environment:angularApiUrl}/classes/igxhintdirective.html) directive.
+
+The following example uses two-way data binding and demonstrates how to inspect the control's state by exporting the `ngModel` to a local variable.
```

Comment:
Also include an example on how to inspect general form validity through the ngForm. In general the check there would be perform in the submit button to make it enabled/disabled.